Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM cimg/base:current
+
+ARG OTP_VERSION
+ARG TARGETARCH
+ENV OTP_VERSION=$OTP_VERSION
+ENV ARCH=$TARGETARCH
+
+RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
+    libncurses5-dev \
+    unixodbc-dev
+
+RUN --mount=type=bind,source=builds/erlang-$TARGETARCH.tar.gz,target=erlang-$TARGETARCH.tar.gz \
```

Comment:
The problem is that every run command forms a separate layer, so we need to remove the source code in the same step that adds it - otherwise image size would be huge.